modules/sdk-coin-dot/src/lib/iface.ts

```diff
@@ -142,12 +157,73 @@ export interface AddAnonymousProxyArgs {
  */
 export type BatchCallObject = {
   callIndex: string;
-  args: Record<string, any>;
+  args:
+    | Record<string, any>
```

Comment:
nit: would love to see `any` go away; doesn't have to be in this PR since its not the scope